### DISCUSSION TITLE
Allow installing as package despite new input system

### DISCUSSION THREAD

**Discussion by NoTuxNoBux** [yasirkula/UnityIngameDebugConsole#65] [Ideas]
6123ac72027a0cc2355465562ce598b96079351c added support for the new input system, and this package can already be installed as an actual Unity package through `manifest.json`, but the combination is unfortunately not possible according to the README if only the new input system is used. I was wondering if this is a planned feature, since the new input system is quickly moving to replace the older one in recent Unity versions as well as 2020 LTS.

Is the technical limitation currently that an additional assembly reference has to be added in order for it to activate properly? If so, and it's not (easily) fixable, on the long term it might be more interesting to reverse the defaults, e.g. in a new major release such as 2.0, and to ask users to install manually if they want the legacy input system instead?

**Comment by yasirkula** [comment]
Thank you for the suggestion! I haven't used the new Input System myself and would like to know approx. what percentage of Unity users use the new Input System before I decide to make this change. Don't know if there are any polls/charts for this statistic yet.
